site/docs/guides/reverse-proxy.md

```diff
@@ -0,0 +1,161 @@
+# Nessie behind a Reverse Proxy
+
+If you run Nessie behind a Reverse Proxy like istio or nginx, both the reverse proxy and Nessie/Quarkus need to be
+configured accordingly.
+
+!!! warn
+    Make sure to understand what the configuration options mean, before you apply those to any production setup!
+
+!!! info
+    Passing the `X-Forwarded-Proto`, `X-Forwarded-Host` and `X-Forwarded-Port` headers to Nessie works for
+    Iceberg REST.
+
+The follow config options are mentioned only for documentation purposes. Consult the
+[Quarkus documentation](https://quarkus.io/guides/http-reference#reverse-proxy) for "Running behind a reverse proxy"
+and configure those depending on your actual needs.
+```properties
+quarkus.http.proxy.proxy-address-forwarding=true
+quarkus.http.proxy.allow-x-forwarded=true
```

Comment:
Quarkus proxy docs make the impression that the standardized `Forwarded` header may be preferred. However, it does not provide the same level of [functionality](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded) as the `X-Forwarded-*` headers together (specifically port and prefix cannot be defined in `Forwarded`) . I think it might be worth adding a paragraph to clarify that. WDYT?